src/EventListener/DataContainer/LeadExportListener.php

```diff
@@ -208,6 +208,8 @@ private function updatePalette($dc = null): void
     private function loadJsAndCss()
     {
         $GLOBALS['TL_JAVASCRIPT'][] = $GLOBALS['BE_MOD']['leads']['lead']['javascript'];
-        $GLOBALS['TL_CSS'][] = $GLOBALS['BE_MOD']['leads']['lead']['stylesheet'];
+        if (isset($GLOBALS['BE_MOD']['leads']['lead']['stylesheet'])) {
+            $GLOBALS['TL_CSS'][] = $GLOBALS['BE_MOD']['leads']['lead']['stylesheet'];
```

Comment:
I'm not sure if we could even drop that, because `stylesheet` is not defined anymore!

Reply:
please drop it then 😊